server/src/main/java/org/opentosca/toscana/plugins/cloudformation/CloudFormationFileCreator.java

```diff
@@ -141,6 +126,54 @@ private void createFileUploadScript() throws IOException {
         }
     }
 
+    /**
+     Creates the script for creating the CloudFormation stack from the template.
+     */
+    private void createCreateStackScript() throws IOException {
+        logger.debug("Creating create-stack script.");
+        BashScript createStackScript = new BashScript(cfnModule.getFileAccess(), FILENAME_CREATE_STACK);
+
+        // Build deploy command
+        StringBuilder deployCommand = new StringBuilder("");
+        deployCommand.append(CLI_COMMAND_CREATESTACK + CLI_PARAM_STACKNAME)
+            .append(cfnModule.getStackName()).append(" ")
+            .append(CLI_PARAM_TEMPLATEFILE).append("../").append(TEMPLATE_YAML);
+
+        // Add IAM capability if needed
+        List<String> filesToBeUploaded = cfnModule.getFilesToBeUploaded();
+        if (!filesToBeUploaded.isEmpty()) {
+            logger.debug("Adding IAM capability to create stack command.");
+            deployCommand.append(" " + CLI_PARAM_CAPABILITIES + " " + CAPABILITY_IAM);
+        }
+
+        // Add parameters if needed
+        Map<String, Parameter> parameters = cfnModule.getParameters();
+        if (!parameters.isEmpty()) {
+            logger.debug("Adding parameters to create stack command.");
+            deployCommand.append(" " + CLI_PARAM_PARAMOVERRIDES);
+            for (Map.Entry<String, Parameter> entry : parameters.entrySet()) {
+                String id = entry.getKey();
+                deployCommand.append(" ").append(id).append("=$").append(id).append("Var");
+            }
+        }
+
+        deployCommand.append(" &");
+        createStackScript.append(deployCommand.toString());
+    }
+
+    /**
+     Creates a script to delete the S3Bucket and the deployed Stack.
+     Note: May or may not be included in the final version. Currently used for quicker manual debugging.
+     */
+    private void createCleanUpScript() throws IOException {
```

Comment:
Simmilar to above. `create` sounds a bit wrong, because you actually write/copy these files in the working directory